roles/cobbler/templates/kickstarts/cephlab_ubuntu.preseed

```diff
@@ -110,7 +110,9 @@ d-i pkgsel/upgrade select safe-upgrade
 d-i pkgsel/update-policy select none
 
 # Install GRUB to default location.  Required as of Xenial.
+#if $os_version == 'xenial'
```

Comment:
I mimicked the existing `if` statements but you're right.. this particular check should probably be based on a numerical major version to future-proof it.